src/frontend/staff-portal/src/app/components/ticket-request/ticket-request.component.ts

```diff
@@ -1,20 +1,81 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, Input } from '@angular/core';
+import { FormBuilder, FormGroup } from '@angular/forms';
+import { Dispute } from 'app/api';
+import { ViolationTicketService } from 'app/services/violation-ticket.service';
 
 @Component({
   selector: 'app-ticket-request',
   templateUrl: './ticket-request.component.html',
   styleUrls: ['./ticket-request.component.scss']
 })
 export class TicketRequestComponent implements OnInit {
+  @Input() disputeInfo: Dispute;
   public collapseObj: any = {
     contactInformation: true
- }
-  constructor() { }
+  }
+  public form: FormGroup;
+
+  constructor(
+    protected formBuilder: FormBuilder,
+    private violationTicketService: ViolationTicketService
+  ) {
+    this.form = this.formBuilder.group({
+      timeToPayReason: null,
+      fineReductionReason: null,
+      disputedCount1: this.formBuilder.group({
+        plea: null,
+        requestType: null,
+        appearInCourt: null,
+        section: null,
+        description: null,
+        ticketedAmount: null
+      }),
+      disputedCount2: this.formBuilder.group({
+        plea: null,
+        requestType: null,
+        appearInCourt: null,
+        section: null,
+        description: null,
+        ticketedAmount: null
+      }),
+      disputedCount3: this.formBuilder.group({
+        plea: null,
+        requestType: null,
+        appearInCourt: null,
+        section: null,
+        description: null,
+        ticketedAmount: null
+      }),
+    });
+  }
+
```

Comment:
this.disputedCountForms = [];
Then create the form and push into it later

this.disputeInfo.disputedCounts.forEach(disputedCount => {
...
...
...
})